spikeinterface/core/base.py

```diff
@@ -422,7 +426,7 @@ def load(file_path):
                     d = pickle.load(f)
             else:
                 raise ValueError(f'Impossible to load {file_path}')
-            extractor = BaseExtractor.from_dict(d)
+            extractor = BaseExtractor.from_dict(d, file_path.parent)
```

Comment:
Here I think we need something to check if it is abslute or not.
We need to be flexible to handle both relative and absolite.

Reply:
It checks with exists